lbr_app_node:
  ros__parameters:
    joint_velocity_command_limit: [2.0, 2.0, 2.0, 2.0, 2.0, 2.0, 2.0]
    wrench_command_limit: [1.0, 1.0, 1.0, 0.5, 0.5, 0.5]
    torque_command_limit: [1.0, 1.0,  1.0,  1.0,  1.0,  1.0,  1.0]
